vncviewer/CConn.h

```diff
@@ -76,6 +82,8 @@ class CConn : public rfb::CConnection
   void handleClipboardAnnounce(bool available) override;
   void handleClipboardData(const char* data) override;
 
+  void reset_password_data();
```

Comment:
Please change this to `resetPassword()` as we use different naming style for methods and plain functions.